exporters/otlp/options.go

```diff
@@ -43,3 +43,22 @@ func WithMetricExportKindSelector(selector metricsdk.ExportKindSelector) Exporte
 		cfg.exportKindSelector = selector
 	}
 }
+
+// ProtocolDriverOption provides options for setting up a split driver
+type ProtocolDriverOption func(d *splitDriver)
```

Comment:
```suggestion
type SplitDriverOption func(d *splitDriver)
```

I'm wondering if this name makes more sense as these options are only applicable to the `splitDriver`.

Reply:
Agreed and updated.